web_server/users/templates/users/login.html

```diff
@@ -1,26 +1,51 @@
-{% extends "index.html" %}
-{% load crispy_forms_tags %}
-{% block content %}
-<div class="container m-3">
-    <form method="POST">
-        {% csrf_token %}
-        <fieldset class="form-group">
-            <legend class="border-bottom mb-4">Log In Now!</legend>
-            {{form | crispy}}
-        </fieldset>
-        <div class="form-group">
-            <button class="btn btn-outline-info" type="submit">
-                Log In
-            </button>
-        </div>
-    </form>
-    <div class="border-top pt-3">
-        <small class="text-muted">
-            Need An Account ? <a class="ml-2" href="{% url 'register' %}">Sign Up Now</a>
-        </small>
-
-    </div>
 
+{% load static %}
+{% load crispy_forms_tags %}
+<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+	integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+	crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+	integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+	integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+	crossorigin="anonymous"></script>
+<link rel="stylesheet" href="{% static '/users/css/style.css' %}">
+<script src='https://cdn.jsdelivr.net/npm/vue'></script>
+<script src='https://cdnjs.cloudflare.com/ajax/libs/d3/5.9.1/d3.min.js'></script>
```

Comment:
I'm also seeing Vue and D3 as dependencies here. Are these just for the mandala animation on this page?